src/main/scala/com/github/unchama/seichiassist/listener/PlayerInventoryListener.scala

```diff
@@ -613,5 +615,56 @@ class PlayerInventoryListener(implicit effectEnvironment: EffectEnvironment,
     } else if (topinventory.getTitle == LIGHT_PURPLE.toString + "" + BOLD + "スキルを進化させました") {
       event.setCancelled(true)
     }
+
+  }
+
+
+
+  @EventHandler
+  /**
+   * 名義除去システム
+   */
+  def onItemNameRemoveEvent(event: InventoryCloseEvent): Unit = {
+    val player = event.getPlayer match {
+      case p: Player => p
+      case _ => return
+    }
+    val inventory = event.getInventory
+
+    //インベントリサイズが36でなければ処理を終了させる
+    if (inventory.row != 4){
+      return
+    }
+    if (inventory.getTitle == s"$GOLD${BOLD}名義をなくしたいアイテムを投入してください") {
+      val items = inventory.getContents
+
+      var count = 0
+      //for文を使い、1つずつアイテムを見ていく
+      for(item <- items){
+        if(item != null){
+          if (item.getItemMeta.hasLore && item.getItemMeta.hasLore){
+            val itemstack= item
+            if (Util.itemStackContainsOwnerName(itemstack:ItemStack, player.getName)){
+              val itemLore = item.getItemMeta.getLore.asScala.toList
+              //itemLoreのListの中から、"所有者"で始まるものを弾き、新しく「所有者:なし」を付け加えたLoreをアイテムにつける
+              val removedNameLore = itemLore.filterNot(n => n.startsWith("所有者"))
+              val newLore = removedNameLore.::("所有者:なし").asJava
+              //ついているitemLoreをNilに置き換え、そこからまたNewLoreをセットする
+              itemLore.map(loreElement => Nil).foldLeft(Nil: List[Int])(_ ++ _)
+              item.getItemMeta.setLore(newLore)
+            }
+          }
+         Util.grantItemStacksEffect[IO]()
```

Comment:
ここで括弧の中に何も指定しないと、何のアイテムも付与されません
```suggestion
         Util.grantItemStacksEffect[IO](item)
```